tests/eosio.msig_tests.cpp

```diff
@@ -484,6 +484,26 @@ BOOST_FIXTURE_TEST_CASE( update_system_contract_all_approve, eosio_msig_tester )
                   ("requested", perm)
    );
 
+   produce_blocks();
+   produce_blocks();
+
+   // verify the proposal 
+   const vector<char> proposal_first_bin = get_row_by_account( "eosio.msig"_n,  "alice"_n, "proposal"_n, "first"_n );
+   const auto proposal_first_result = abi_ser.binary_to_variant("proposal", 
+                                                                proposal_first_bin, 
+                                                                abi_serializer::create_yield_function( abi_serializer_max_time ) );
+   /*  
+   const auto proposal_first_json = fc::json::to_pretty_string(proposal_first_result);
+   std::cout << proposal_first_json << std::endl;
+   */
```

Comment:
I think it's good that we have these prints statements for future reference.
Instead, can we uncomment them and convert them to `ilog` statements? This will give the user the option to print stuff out if they choose to do so without having to uncomment the prints and recompile.

We'll have to include:
`#include <fc/log/logger.hpp>`

Example of `ilog` in one of the tests:
https://github.com/EOSIO/eosio.contracts/blob/f6578c45c83ec60826e6a1eeb9ee71de85abe976/tests/eosio.system_tests.cpp#L1794

Reply:
Good comments, added the log, below is what being printed:
proposal first approval in json:
{
  "version": 1,
  "proposal_name": "first",
  "requested_approvals": [],
  "provided_approvals": [{
      "level": {
        "actor": "alice",
        "permission": "active"
      },
      "time": "2020-01-01T00:00:29.500"
    },{
      "level": {
        "actor": "bob",
        "permission": "active"
      },
      "time": "2020-01-01T00:00:30.000"
    },{
      "level": {
        "actor": "carol",
        "permission": "active"
      },
      "time": "2020-01-01T00:00:30.500"
    }
  ]
}